### PR TITLE
Tweak warning to make more sense to 3rd parties

### PR DESCRIPTION
**Pull request by danielrbradley**
As suggested during work on the TF boilerplate: https://github.com/pulumi/pulumi-tf-provider-boilerplate/pull/135#discussion_r1832763213

### CODE REVIEW

File: provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml [hunk 1]
```diff
@@ -14,29 +14,29 @@ jobs:
...
-        git grep -l 'go:embed' -- provider | xargs sed -i 's/go:embed/ goembed/g'
-    - name: prepare upstream
-      continue-on-error: true
-      run: make upstream
-    - name: golangci-lint
-      uses: golangci/golangci-lint-action@v6
-      with:
-        version: v1.60
-        working-directory: provider
+      - name: Checkout Repo
```
Comment (guineveresaenger) [inline provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml:17]: These feel like spurious changes caused by editor opinion.

Comment (danielrbradley) [reply provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml:17]: Bah, yes

Comment (danielrbradley) [reply provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml:17]: Ah - only reset the changes in bridged-provider but missed the other sub-templates

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
I'd rather not flip back and forth with regard to indentation in the templates. We should standardize/enforce via lint the indentation we want. I can be convinced that the changes proposed are "more" correct but it feels like adding unrelated noise to this pull request.

File: provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml (hunk 1 above)
Comment (danielrbradley) [reply provider-ci/internal/pkg/templates/provider/.github/workflows/lint.yml]: Fixed

**Comment by danielrbradley** [comment]
> I'd rather not flip back and forth with regard to indentation in the templates. We should standardize/enforce via lint the indentation we want. I can be convinced that the changes proposed are "more" correct but it feels like adding unrelated noise to this pull request.

This is just an artifact of how our templates are currently oddly structured. Running find & replace in the editor causes an auto-format which then 'fixes' all of these indents which I then have to manually reset 😫

**Review by guineveresaenger — APPROVED** [review]
Thank you for cleaning this up!

**Pull request merged**